build/pubspec.yaml

```diff
@@ -21,3 +21,9 @@ dev_dependencies:
   build_test: ^2.0.0
   lints: '>=1.0.0 <3.0.0'
   test: ^1.16.0
+
+dependency_overrides:
```

Comment:
Do you need these?

Reply:
I need them for latest analyzer to resolve for tests. I was going to drop when I publish.